### DISCUSSION TITLE
Migration to 15.0.2 issues

### DISCUSSION THREAD

**Discussion by A380Coding** [jomjol/AI-on-the-edge-device#2100] [General] [answered]
I updated 14.0.3 to 15.0.2. On two ESP a lot of parameters are disabled with no value. Any idea what causes this?
I checked the config manually and compared but do not find any useful information.

**Comment by A380Coding** [comment]
Just found out it seems to be a browser cache issue. Leaving this post here in case somebody else has the same issue.
-> Restart browser or use private mode.

**Comment by A380Coding** [comment]
Values for DecimalShift, AnalogDigitalTransitionStart, MAxRateValue, MaxRateType are disabled on all my ESP32. They can be configured in the config.ini but values will not be shown in GUI

**Reply by A380Coding** [reply]
AlowNegative rates is set to false in config.ini, but in gui it shows true

**Reply by caco3** [reply]
can you provide your config.ini so we can reproduce it?

As you already found out, caching also can be an issue.

**Reply by A380Coding** [reply]
I formated the sd card and did a fresh setup, but this issue is still present, other parameters reapeared after refreshing cache (did not keep that in mind in first place)

`
[TakeImage]
;RawImagesLocation = /log/source
WaitBeforeTakingPicture = 5
;RawImagesRetention = 15
Demo = false
Brightness = 0
Contrast = 0
Saturation = 0
LEDIntensity = 10
ImageQuality = 10
ImageSize = VGA
FixedExposure = false

[Alignment]
InitialRotate = 0
InitialMirror = false
SearchFieldX = 20
SearchFieldY = 20
AlignmentAlgo = highAccuracy
FlipImageSize = false
/config/ref0.jpg 44 386
/config/ref1.jpg 350 386

[Digits]
Model = /config/dig-class100-0150_s2_q.tflite
CNNGoodThreshold = 0.5
;ROIImagesLocation = /log/digit
;ROIImagesRetention = 3
main.dig1 80 403 30 59 false
main.dig2 127 403 30 59 false
main.dig3 170 403 30 59 false
main.dig4 216 403 30 59 0
main.dig5 262 403 30 59 0
main.dig6 310 403 30 59 0
main.dig7 352 403 30 59 0

;[Analog]
Model = /config/ana-cont_11.3.1_s2.tflite
;ROIImagesLocation = /log/analog
;ROIImagesRetention = 3
main.ana1 432 230 92 92 false
main.ana2 379 332 92 92 false
main.ana3 283 374 92 92 false
main.ana4 155 328 92 92 false

[PostProcessing]
main.DecimalShift = -1
main.AnalogDigitalTransitionStart = 9.2
PreValueUse = false
PreValueAgeStartup = 60
main.AllowNegativeRates = false
;main.MaxRateValue = 5
;main.MaxRateType = AbsoluteChange
main.ExtendedResolution = false
main.IgnoreLeadingNaN = false
ErrorMessage = true
CheckDigitIncreaseConsistency = true

;[MQTT]
;Uri = mqtt://IP-ADRESS:1883
;MainTopic = watermeter
;ClientID = watermeter
;user = USERNAME
;password = PASSWORD
RetainMessages = false
HomeassistantDiscovery = false
;MeterType = other

;[InfluxDB]
;Uri = undefined
;Database = undefined
;Measurement = undefined
;user = undefined
;password = undefined

;[InfluxDBv2]
;Uri = undefined
;Database = undefined
;Measurement = undefined
;Org = undefined
;Token = undefined
;main.Fieldname = undefined

;[GPIO]
;IO0 = input disabled 10 false false 
;IO1 = input disabled 10 false false 
;IO3 = input disabled 10 false false 
;IO4 = built-in-led disabled 10 false false 
;IO12 = input-pullup disabled 10 false false 
;IO13 = input-pullup disabled 10 false false 
LEDType = WS2812
LEDNumbers = 2
LEDColor = 150 150 150

[AutoTimer]
AutoStart = true
Interval = 5

[DataLogging]
DataLogActive = true
DataFilesRetention = 30

[Debug]
LogLevel = 1
LogfilesRetention = 30

[System]
TimeZone = CET-1CEST,M3.5.0,M10.5.0/3
TimeServer = pool.ntp.org
;Hostname = undefined
;RSSIThreshold = 0
SetupMode = false

`

**Reply by A380Coding** [reply]
![ai](https://user-images.githubusercontent.com/80172816/221967659-923bfb44-ec45-4aa4-98a5-3eab1386778c.png)

**Comment by caco3** [comment, marked as answer]
Please upgrade to https://github.com/jomjol/AI-on-the-edge-device/releases/tag/v15.0.3

When we created  v15.0.2, one change got lost, I added it now again and made v15.0.3
